cli/tests/unit/write_file_test.ts

```diff
@@ -125,6 +139,20 @@ unitTest(
   }
 );
 
+unitTest(
+  { perms: { read: true, write: true } },
+  async function writeFileUrl(): Promise<void> {
+    const enc = new TextEncoder();
+    const data = enc.encode("Hello");
+    const fileUrl = new URL(`file://${Deno.makeTempDirSync()}/test.txt`);
+    await Deno.writeFile(fileUrl, data);
```

Comment:
It will be great if you remove redundant files after testing.

Reply:
Sure thing, should I add that to the existing unit tests in this file as well, or just with the new tests?

Reply:
for more clear PRs and commits, you can interact with other tests in another PR.
for now just care about new cases.